js/models/roomcollection.js

```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```

Comment:
That updates the room participant table ``spreedme_room_participants``. That rooms table does not have that attribute, does it?

(I have not really tested your changes yet, but as far as I know the code, this won't work)